Packs/SymantecMSS/Integrations/SymantecMSS/SymantecMSS.py

```diff
@@ -321,8 +321,11 @@ def query_incident_cmd():
     result = query_incident(demisto.args()["number"], workflow_query=True)
 
     # Create minimal signature list
+    data = result.get("SignatureList", {}).get("Signature", [])
```

Comment:
in case the `Signature` value is `None`
```suggestion
    data = result.get("SignatureList", {}).get("Signature") or []
```